components/function-controller/internal/webhook/conversion_webhook.go

```diff
@@ -277,10 +281,12 @@ func (w *ConvertingWebhook) convertFunctionV1Alpha2ToV1Alpha1(src, dst runtime.O
 
 func (w *ConvertingWebhook) convertSpecV1Alpha2ToV1Alpha1(in *serverlessv1alpha2.Function, out *serverlessv1alpha1.Function) error {
 	out.Spec.Env = in.Spec.Env
-	out.Spec.MaxReplicas = in.Spec.MaxReplicas
-	out.Spec.MinReplicas = in.Spec.MaxReplicas
```

Comment:
It looks like a bug when the wrong field is assigned to minReplicas (should be min -> min, not max -> min) so I decided to change it. Am I right?